iroha_data_model/src/lib.rs

```diff
@@ -301,6 +302,31 @@ pub trait Identifiable: Debug + Clone {
     type Id: Into<IdBox> + Debug + Clone + Eq + Ord;
 }
 
+/// Limits of length of the identifiers (e.g. in [`Domain`], [`NewAccount`], [`AssetDefinition`])
+#[derive(Debug, Clone, Copy, Decode, Encode, Serialize, Deserialize)]
+pub struct LengthLimits {
+    /// Minimal length (inclusive).
+    min_len: u32,
+    /// Maximal length (inclusive).
+    max_len: u32,
+}
+
+impl LengthLimits {
+    /// Constructor.
+    pub const fn new(min_len: u32, max_len: u32) -> Self {
+        Self { min_len, max_len }
+    }
+}
+
+impl From<LengthLimits> for Range<usize> {
```

Comment:
May be it can be changed to [`RangeInclusive`](https://doc.rust-lang.org/std/ops/struct.RangeInclusive.html)

Reply:
Agree. Updated